Loop/Managers/DoseMath.swift

```diff
@@ -234,7 +234,9 @@ extension Collection where Element: GlucoseValue {
         at date: Date,
         suspendThreshold: HKQuantity,
         sensitivity: HKQuantity,
-        model: InsulinModel
+        model: InsulinModel,
+        insulinOnBoard: Double? = nil,
```

Comment:
If Loop will be enforcing a cap on IOB in all cases, these parameters should not be optional, and the tests should be updated to pass values in.

Reply:
This way we can see that existing tests pass, and if the results change, how they change.

Reply:
I updated the DoseMathTests to pass in the parameters.
Then I reversed the change I made yesterday that made the parameters optional. I hope this is what you meant.